Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -319,24 +327,22 @@ var ListView = React.createClass({
       }
     }
 
-    var props = merge(
-      this.props, {
-        onScroll: this._onScroll,
-        stickyHeaderIndices: sectionHeaderIndices,
-      }
-    );
+    var {
+      renderScrollView,
+      ...props,
+    } = this.props;
     if (!props.scrollEventThrottle) {
       props.scrollEventThrottle = DEFAULT_SCROLL_CALLBACK_THROTTLE;
     }
 
-    return (
-      <ScrollView {...props}
-        ref={SCROLLVIEW_REF}>
-        {header}
-        {bodyComponents}
-        {footer}
-      </ScrollView>
-    );
+    var scrollView = renderScrollView();
+    return React.cloneElement(scrollView, {
+      ...props,
+      ...scrollView.props,
```

Comment:
I want to merge the ListView's props (e.g. scrollEventThrottle) with the underlying ScrollView's props, if renderScrollView returned something like `<ScrollView pageSize={10} />` but the implementation in this diff is not flexible enough. I'll update it to your `renderScrollView(props)` idea  after I try composing some ScrollViews together.